src/algebra/category/CommRing/colimits.lean

```diff
@@ -4,6 +4,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
 import algebra.category.CommRing.basic
+import category_theory.limits.concrete_category
```

Comment:
Did you mean to include these imports in this PR?

Reply:
Yes, because in `CommRing.basic` I reduce the import from `Group.default` to `Group.basic`, some additional imports are needed here.